diff --git a/Android.bp b/Android.bp
index d175c6d..7c983de 100644
--- a/Android.bp
+++ b/Android.bp
@@ -51,27 +51,6 @@ rust_proc_macro {
     ],
 }
 
-rust_test_host {
-    name: "remain_test_src_lib",
-    crate_name: "remain",
-    cargo_env_compat: true,
-    cargo_pkg_version: "0.2.2",
-    srcs: ["src/lib.rs"],
-    test_suites: ["general-tests"],
-    auto_gen_config: true,
-    test_options: {
-        unit_test: true,
-    },
-    edition: "2018",
-    rustlibs: [
-        "libproc_macro2",
-        "libquote",
-        "libsyn",
-        "libtrybuild",
-    ],
-    proc_macros: ["librustversion"],
-}
-
 rust_defaults {
     name: "remain_test_defaults",
     crate_name: "remain",
@@ -84,7 +63,6 @@ rust_defaults {
         "libproc_macro2",
         "libquote",
         "libsyn",
-        "libtrybuild",
     ],
     proc_macros: [
         "libremain",
@@ -92,15 +70,6 @@ rust_defaults {
     ],
 }
 
-rust_test_host {
-    name: "remain_test_tests_compiletest",
-    defaults: ["remain_test_defaults"],
-    srcs: ["tests/compiletest.rs"],
-    test_options: {
-        unit_test: true,
-    },
-}
-
 rust_test_host {
     name: "remain_test_tests_stable",
     defaults: ["remain_test_defaults"],
